### ISSUE TITLE
Cannot Build app after moving to latest version

### ISSUE THREAD

**Issue by BJRichardsAZ** [robodigital/cobrowseio_flutter#4]
I'm not sure exactly why, but the app fails to compile now after moving the version number up on both IOS and Android.
Android: 
<img width="492" alt="image" src="https://user-images.githubusercontent.com/72472725/154314510-a2bdeb18-6aa3-4662-9f3b-d94ca9cde997.png">
IOS: 
<img width="490" alt="image" src="https://user-images.githubusercontent.com/72472725/154315516-a0738f8c-dce0-4b2d-8cda-883978c6a14a.png">

**Comment by BJRichardsAZ** [comment]
I see some information online that says our android issue may just be our jdk version; however I'm unsure of what to do about the xcode issue

**Comment by robinbonnes** [comment]
I did apply some fixes for both. Can you try updating from this git repo by changing in your `pubspec.yaml`:

```
  cobrowseio_flutter:
    git:
      url: https://github.com/robodigital/cobrowseio_flutter
```

Then run: `flutter pub upgrade`

Could you let me know if it is working for iOS?

**Comment by BJRichardsAZ** [comment]
Looks like it is now building on IOS!

**Comment by robinbonnes** [comment]
Great, published the new version `1.0.2`
